integration_tests/src/utils.rs

```diff
@@ -131,6 +135,40 @@ pub fn get_eth_node() -> String {
     format!("http://{}:8545", NODE_IP)
 }
 
+pub fn get_altheadb_contract_addr() -> Address {
+    "0xb9b674D720F96995ca033ec347df080d500c2230"
```

Comment:
this hardcoded address won't always be correct, in fact I'm not sure it would even work twice, (it might if all conditions on the testnet chain are exactly repeatable, but I wouldn't bet on it) 

you should parse the contract address from the deployment script. I'm sure there's an example in the gravity repo.

Reply:
done